Tasks/AppCenterDistributeV3/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -29,9 +29,9 @@
   "loc.input.help.releaseNotesFile": "Select a UTF-8 encoded text file which contains the Release Notes for this version.",
   "loc.input.label.isMandatory": "Require users to update to this release",
   "loc.input.label.destinationType": "Release destination",
-  "loc.input.label.destinationGroupIds": "Distribution Group IDs",
+  "loc.input.label.destinationGroupIds": "Destination IDs",
```

Comment:
Are these changes intentional?

Reply:
@DennisPan this file is auto-generated.  Since the file is changed in this PR, that means someone changed this string in `task.json` but didn't also change the string in this file (`resources.resjson`) in a previous PR.